rocketpy/rocket/aero_surface/fins/fins.py

```diff
@@ -423,7 +308,7 @@ def compute_forces_and_moments(
             * omega[2]
             / 2
         )
-        M3 = M3_forcing - M3_damping
+        M3 = M3_forcing + M3_damping
         return R1, R2, R3, M1, M2, M3
 
     def to_dict(self, include_outputs=False):
```

Comment:
As both `Fins` and `TrapezoidalFinsMixin` have the `to_dict` method, which method will the `TrapezoidalFins` class use?